en/development/configuration.rst

```diff
@@ -282,6 +282,21 @@ data back::
 
 If $key is left null, all values in Configure will be returned.
 
+.. versionadded:: 3.1.7
+
+.. php:staticmethod:: readOrFail($key)
+
+Reads configuration data just like :php:method:`Cake\\Core\\Configure::read` but expects to find a key/value pair.
+In case the requested pair does not exist, a :php:class:`RuntimeException` will be thrown.
+
```

Comment:
I fixed it back to how it is a few lines above. Someone told me to remove `::` before. Working like this?